src/latex/raw.ml

```diff
@@ -111,6 +111,12 @@ let emph pp = create "emph" pp
 
 let bold pp = create "bold" pp
 
+let left pp = create "left" pp
+
+let center pp = create "center" pp
+
+let right pp = create "right" pp
```

Comment:
I don't thing `left` and `right` exist. (I'm not a latex expert)
I think you'll need to use `flushleft`, `center` and `flushright` and to call them with `env` (see `verbatim` above)

Reply:
Yep, `flushleft` and `flushright` are right. thanks 🙂!